docker/dockerTemplate.sh

```diff
@@ -13,14 +12,21 @@ help() {
     echo "-c : clean (create the tagged template file). To be executed after init"
     echo "-v : the version to build"
     echo "-t : the target to build"
+    echo "-p : the parent version to user"
```

Comment:
Nit: `user` -> `use`

Reply:
Fixed in 9abbde23139f9f89345e2f9dab6df251d67d7d4e.